src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -163,51 +175,55 @@ export default class GitOpsConfiguration extends Component<GitOpsProps, GitOpsSt
                 <div className="login__sso-flex">
                     <div>
                         <label className="tertiary-tab__radio">
-                            <input type="radio" name="status" value={GitProvider.GitLab} checked={this.state.tab === "gitlab"} onChange={this.handleGitopsTab} />
+                            <input type="radio" name="status" value={GitProvider.Github} checked={this.state.tab === "github"} onChange={this.handleGitopsTab} />
                             <span className="tertiary-tab sso-icons">
-                                <aside className="login__icon-alignment"><GitLab /></aside>
-                                <aside className="login__text-alignment"> GitLab</aside>
+                                <aside className="login__icon-alignment"><GitHub /></aside>
+                                <aside className="login__text-alignment"> GitHub</aside>
                                 <label>
```

Comment:
Label tag is used to focus on form field by wrapping input tag. Also, don't use label inside label. div tag is better option here.